datasets/crass/README.md

```diff
@@ -0,0 +1,171 @@
+---
+annotations_creators:
+- crowdsourced
+language_creators:
+- crowdsourced
+- expert-generated
+- machine-generated
+languages:
+- en-US
+licenses:
+- apache-2.0
+multilinguality:
+- monolingual
+pretty_name: CRASS
+size_categories:
+- unknown
+source_datasets:
+- original
+task_categories:
+- text-classification
+task_ids:
+- natural-language-inference
+---
+
+# Dataset Card for CRASS
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://www.crass.ai
+- **Repository:** https://github.com/apergo-ai/CRASS-data-set
+- **Paper:** [Needs More Information]
+- **Leaderboard:** https://www.crass.ai/ledger.php
+- **Point of Contact:** j.frohberg@apergo.net
+
+### Dataset Summary
+
+The data consists of so-called PCTs (Premise-Counterfactual Tuples). They contrast a hypothetical situation using a counterfactual conditional against a base premise. In the fixed target mode there is one correct answer and two alternative incorrect answers.
+
+### Supported Tasks and Leaderboards
+
+zero-shot-classification: The dataset can be used to benchmark a model for zero-shot or few-shot classification tasks, which consists in selecting the correct answer out of three possible answers. Success on this task is typically measured by achieving a high T1 accuracy. A [BART-based model] (https://huggingface.co/facebook/bart-large-mnli) model trained on MNLI currently achieves a [T1 accuracy of 0.48]. GPT-3 family accessed via OpenAI beta API using davinchi model currently achieves a [T1 accuracy of 0.56].
+
+### Languages
+
+en-US
+
+## Dataset Structure
+
+### Data Instances
+
+  {
+    "PCTID": 6,
+    "BatchID": 1,
+    "Premise": "A woman sees a fire.",
+    "QCC": "What would have happened if the woman had touched the fire?",
+    "CorrectAnswer": "She would have been burned.",
+    "Answer1": "She would not have been burned.",
+    "Answer2": "That is not possible.",
+    "PossibleAnswer3": "She would have seen fire.",
+    "canary_string": ""
+}
```

Comment:
````suggestion
```
{
  "PCTID": 6,
  "BatchID": 1,
  "Premise": "A woman sees a fire.",
  "QCC": "What would have happened if the woman had touched the fire?",
  "CorrectAnswer": "She would have been burned.",
  "Answer1": "She would not have been burned.",
  "Answer2": "That is not possible.",
  "PossibleAnswer3": "She would have seen fire.",
  "canary_string": ""
}
```
````